src/haddock/libs/libio.py

```diff
@@ -35,6 +37,101 @@ def read_from_yaml(yaml_file):
     return ycfg
 
 
+def write_dic_to_file(
+        data_dict,
+        output_fname,
+        info_header="",
+        sep="\t",
+        ):
+    """
+    Write a dictionary to a file.
```

Comment:
To what I understand, this function does more than write a dictionary to a file. To simply write a dict to a file there's `json.dump` function already. I think it is something more like: "Create a table from a dictionary?"